tests/models/test_image_querying.py

```diff
@@ -507,8 +507,8 @@ def test_find_images(ptf_reference_image_datastores, ptf_ref,
     assert all(im.instrument == 'PTF' for im in found6)
     assert set( f.id for f in found6 ) == set( f.id for f in found1 )
 
-    found7 = Image.find_images(filter='r DECam SDSS c0002 6415.0 1480.0')
-    assert all(im.filter == 'r DECam SDSS c0002 6415.0 1480.0' for im in found7)
+    found7 = Image.find_images(filter='r')
+    assert all(im.filter == 'r' for im in found7)
```

Comment:
This works fine for this test, for now, however since we are now working with short filter names, would it be better to make this query search for "filter r AND instrument DECam"?

Reply:
For purposes of this test, I think this is fine.  The database should be clean and only have what we expect in it.

For real production use, yeah, you want to filter on both instrument and filter.